packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -0,0 +1,47 @@
+export default function globalAssignmentToDefaultExport({source}, {jscodeshift: j}, {printOptions = {}, appGlobalIdentifiers}) {
+  function findFirstMember(memberExpression) {
+    if (j.MemberExpression.check(memberExpression)) {
+      return findFirstMember(memberExpression.object);
```

Comment:
We do, the `findFirstMember` is just for validating that it's a global variable assignment that we care about.

`Sello.Components.Component = Component`

would get converted to `export default Component`. I guess we could also go through the scope and find where `Component` is defined in that case, but I don't really care that much